packages/amplify-e2e-tests/package.json

```diff
@@ -17,63 +17,49 @@
   ],
   "private": true,
   "scripts": {
-    "e2e": "npm run setup-profile && jest",
+    "e2e": "npm run setup-profile && jest --verbose",
     "setup-profile": "ts-node ./src/configure_tests.ts"
   },
   "dependencies": {
+    "@types/jest": "^25.1.3",
```

Comment:
It is not needed based on @SwaySway PR what we just merged and should be in devdeps if would be needed.